contrib/auctex/packages.el

```diff
@@ -12,36 +13,36 @@
     :defer t
     :config
     (progn
-      (defun auctex/build-view ()
-        (interactive)
-        (progn
-          (let ((TeX-save-query nil))
-            (TeX-save-document (TeX-master-file)))
-          (setq build-proc (TeX-command "LaTeX" 'TeX-master-file -1))
-          (set-process-sentinel  build-proc  'auctex/build-sentinel))
-        (TeX-view))
-
-      (defun auctex/build-sentinel (process event)
-        (if (string= event "finished\n")
-            (TeX-view)
-          (message "Errors! Check with C-`")))
-
-      (add-hook 'LaTeX-mode-hook '(lambda () (local-set-key (kbd "H-r") 'auctex/build-view)))
```

Comment:
Yeh, I thought that the layer should really just take care of <kbd>SPC m ...</kbd> keys, and <kbd>H r</kbd> didn't really seem to fit in very well.